src/components/calculator/index.js

```diff
@@ -12,56 +11,94 @@ export const packEV = async (set) => {
     let uncommons = 0;
     let rares = 0;
     let mythics = 0;
+    let basicLands = 0;
     for (let card of setData) {
-        if (card.rarity === 'common') {
+        if (card.type_line.toLowerCase().includes('basic') || card.type_line.toLowerCase().includes('gate')) {
+            basicLands++;
+        } else if (card.rarity === 'common') {
             commons++;
         } else if (card.rarity === 'uncommon') {
             uncommons++;
         } else if (card.rarity === 'rare') {
             rares++;
-        } else { mythics++; }
+        } else if (card.rarity === 'mythic') {
+            mythics++;
+        }
     }
```

Comment:
Could possibly extract this into a function called 'countSetCards' or something that returns the counts in an object

{ uncommons: 34, rares: 34, mythics:10 } etc...

Reply:
going to work on doing this soon when I expand my calculations.